project/CassandraSparkBuild.scala

```diff
@@ -184,7 +198,8 @@ object Dependencies {
   val metrics = Seq(Metrics.metricsCore, Metrics.metricsJson)
 
   val testKit = Seq(Test.akkaTestKit, Test.commonsIO, Test.junit,
```

Comment:
I think we are going to want to change the style on lists like this, @helena was suggestting 
 
    val testKit = Seq(
        elements, elements, elements
        more elements,
       )

And I think that looks pretty good as well